data_structures/src/mainnet_validations.rs

```diff
@@ -72,29 +72,40 @@ impl TapiEngine {
 
     pub fn initialize_wip_information(&mut self) -> (Epoch, HashSet<String>) {
         // Hardcoded information about WIPs
-        let mut bits = vec![vec![]; 32];
+        // TODO: Include first and second hard fork as activated wips
+        //self.wip_activation.insert("first_hard_fork".to_string(), FIRST_HARD_FORK);
+        //self.wip_activation.insert("second_hard_fork".to_string(), SECOND_HARD_FORK);
```

Comment:
Maybe we can use the WIP names?

```suggestion
        self.wip_activation.insert("WIP0008".to_string(), FIRST_HARD_FORK);
        self.wip_activation.insert("WIP0009-0011-0012".to_string(), SECOND_HARD_FORK);
```

Reply:
As @aesedepece said in another PR, I am not sure if we should use the wip numbers as identifiers for the protocol improvements or not, so 🤷🏽

Reply:
But we could put like that and modify in another PR if we consider useful

Reply:
Yes, and maybe we could use a `u32` index instead of a `String`